win32ss/user/user32_vista/dpi.c

```diff
@@ -24,7 +29,7 @@ GetDpiForSystem(VOID)
 {
     HDC hDC;
     UINT Dpi;
-    hDC = GetDC(NULL);
+    hDC = NtUserGetDC(NULL);
```

Comment:
Why this change? GetDC does already the correct thing and is also inlined when being used internally, see:
https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/user/user32/include/ntwrapper.h;hb=2f23e70ad55cc661bc756fb492535fdfab2e9921#l276
(Maybe this is the header you would need to include from within user32_vista, if this is not already done; in regular user32 this header is included via `user32p.h`).

Reply:
No idea why but gcc isn't happy with this at all.
no matter what I revert in this PR GetDC is just.. toast